src/main/kotlin/org/jitsi/nlj/transform/node/PacketLoss.kt

```diff
@@ -22,15 +22,18 @@ import java.util.*
 /**
  * Randomly drops packets at a rate of [lossRate].
  */
-class PacketLoss(private val lossRate: Double) : Node("Packet loss") {
+class PacketLoss(private val lossRate: Double) : TransformerNode("Packet loss") {
```

Comment:
should these be filter?

Reply:
Yes! These were in fact the reason I added the filter, but apparently I forgot to go back and change them.